### DISCUSSION TITLE
How to add line comment

### DISCUSSION THREAD

**Discussion by panderior** [forge/roaster#267] [Q&A] [answered]
When I try to add a line comment to a Java class code I am trying to generate, the roaster library removes the comment and gives just the code.

My intention is to generate a code like this:
public class A {
    public void helper () {
         // this is a comment
         System.out.println("Success");
}}

What have I tried:
* I have tried class and method level JavaDoc, which is a nice feature. But I also want to use line comment.
* I have tried getting the "toUnformattedString" on the JavaClassSource, but the line comments just won't appear.

Your help is very appreciated. Thank you.

**Comment by gastaldi** [comment, marked as answer]
@panderior Thanks for your question. Looks like comments are ignored by the JDT parser somehow, I'm investigating why that is happening. I created #268 to track this

**Reply by panderior** [reply]
Thank you.
